lib/engine/game/g_rolling_stock/step/buy_sell_shares_bid_companies.rb

```diff
@@ -148,7 +148,7 @@ def can_buy?(entity, bundle)
             entity.cash >= @game.next_price_to_right(bundle.corporation.share_price).price
           end
 
-          def buy_shares(entity, bundle, exchange: nil, swap: nil, allow_president_change: true, borrow_from: nil)
+          def buy_shares(entity, bundle, exchange: nil, swap: nil, allow_president_change: true, silent: nil, borrow_from: nil)
```

Comment:
Since the base `buy_shares` has `silent` default to `nil`, does it need to be added to all these subclass overrides? When they call `super` won't the superclass just use the default by default? It'd be nice to avoid touching other game classes if possible.

Reply:
I see I was mistaken with this comment, and the args do need to specified...maybe instead of that, args in the overrides that would default to nil and aren't used in the override can just be sucked in to `**kwargs`?

Reply:
Your commit 0236a8ff5e meant that these changes were no longer required, so they have been removed.